charts/osm/Chart.yaml

```diff
@@ -19,3 +19,6 @@ version: 0.8.2
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
 appVersion: v0.8.2
+
+# This specifies a particular range of k8s versions that the application is compatible with. 
+kubeVersion: >= 1.18.14 <= 1.20.2
```

Comment:
Can we remove the upper bound of 1.20.2 so when a new patch is released, CI with Kind based k8s cluster automatically picks it up?

Reply:
Should we get rid of the upper bound altogether, or set it to the default version for kind used in the CI?

Reply:
We likely want to always support latest supported k8s version including patches, and find issues early, so yes let's get rid of the upper bound. If we cannot support the latest release for whatever reason, we can revisit adding upper bound then.

Also shouldn't the lower bound be 1.18.0.

Reply:
I set it to 1.18.14 because that is currently the default version of k8s on AKS that was used to validate OSM v0.8.2 in our CI in osm-azure (and the kind version in this upstream repo is 1.19.0 I believe) It likely does work for 1.18.0, but since we haven't tested it for v0.8.2 I'm not sure if we should make that the lower bound. 

This does bring up a good question in general of how the lower bound should be determined.

Reply:
nvm, the e2e_k8s_version test validates on multiple versions.